Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -36,15 +36,15 @@ @implementation SentryTimeToDisplayTracker {
     BOOL _waitForFullDisplay;
     BOOL _initialDisplayReported;
     BOOL _fullyDisplayedReported;
-    NSString *_controllerName;
+    NSString *_name;
```

Comment:
I expected this to be related to something with TracingView.name, but I only saw "UIViewController" being passed into the init parameter for this. Did I miss anything?

Reply:
Now with SwiftUI, we also pass the view name.